src/Gui/InputHintPy.cpp

```diff
@@ -0,0 +1,211 @@
+    PyDict_SetItemString(py_constants_dict, "KeyF15", PyLong_FromLong(static_cast<int>(KeyF15)));
+    PyDict_SetItemString(py_constants_dict, "KeyF16", PyLong_FromLong(static_cast<int>(KeyF16)));
+    PyDict_SetItemString(py_constants_dict, "KeyF17", PyLong_FromLong(static_cast<int>(KeyF17)));
+    PyDict_SetItemString(py_constants_dict, "KeyF18", PyLong_FromLong(static_cast<int>(KeyF18)));
+    PyDict_SetItemString(py_constants_dict, "KeyF19", PyLong_FromLong(static_cast<int>(KeyF19)));
+    PyDict_SetItemString(py_constants_dict, "KeyF20", PyLong_FromLong(static_cast<int>(KeyF20)));
+    PyDict_SetItemString(py_constants_dict, "KeyF21", PyLong_FromLong(static_cast<int>(KeyF21)));
+    PyDict_SetItemString(py_constants_dict, "KeyF22", PyLong_FromLong(static_cast<int>(KeyF22)));
+    PyDict_SetItemString(py_constants_dict, "KeyF23", PyLong_FromLong(static_cast<int>(KeyF23)));
+    PyDict_SetItemString(py_constants_dict, "KeyF24", PyLong_FromLong(static_cast<int>(KeyF24)));
+    PyDict_SetItemString(py_constants_dict, "KeyF25", PyLong_FromLong(static_cast<int>(KeyF25)));
+    PyDict_SetItemString(py_constants_dict, "KeyF26", PyLong_FromLong(static_cast<int>(KeyF26)));
+    PyDict_SetItemString(py_constants_dict, "KeyF27", PyLong_FromLong(static_cast<int>(KeyF27)));
+    PyDict_SetItemString(py_constants_dict, "KeyF28", PyLong_FromLong(static_cast<int>(KeyF28)));
+    PyDict_SetItemString(py_constants_dict, "KeyF29", PyLong_FromLong(static_cast<int>(KeyF29)));
+    PyDict_SetItemString(py_constants_dict, "KeyF30", PyLong_FromLong(static_cast<int>(KeyF30)));
+    PyDict_SetItemString(py_constants_dict, "KeyF31", PyLong_FromLong(static_cast<int>(KeyF31)));
+    PyDict_SetItemString(py_constants_dict, "KeyF32", PyLong_FromLong(static_cast<int>(KeyF32)));
+    PyDict_SetItemString(py_constants_dict, "KeyF33", PyLong_FromLong(static_cast<int>(KeyF33)));
+    PyDict_SetItemString(py_constants_dict, "KeyF34", PyLong_FromLong(static_cast<int>(KeyF34)));
+    PyDict_SetItemString(py_constants_dict, "KeyF35", PyLong_FromLong(static_cast<int>(KeyF35)));
+
+    // Mouse Keys
+    PyDict_SetItemString(py_constants_dict, "MouseMove", PyLong_FromLong(static_cast<int>(MouseMove)));
+    PyDict_SetItemString(py_constants_dict, "MouseLeft", PyLong_FromLong(static_cast<int>(MouseLeft)));
+    PyDict_SetItemString(py_constants_dict, "MouseRight", PyLong_FromLong(static_cast<int>(MouseRight)));
+    PyDict_SetItemString(py_constants_dict, "MouseMiddle", PyLong_FromLong(static_cast<int>(MouseMiddle)));
+    PyDict_SetItemString(py_constants_dict, "MouseScroll", PyLong_FromLong(static_cast<int>(MouseScroll)));
+    PyDict_SetItemString(py_constants_dict, "MouseScrollUp", PyLong_FromLong(static_cast<int>(MouseScrollUp)));
+    PyDict_SetItemString(py_constants_dict, "MouseScrollDown", PyLong_FromLong(static_cast<int>(MouseScrollDown)));
+
+    // clang-format on
+
+    py_enum_class = PyObject_CallMethod(py_enum_module, "IntEnum", "sO", name, py_constants_dict);
```

Comment:
By removing the declaration at the top, this could be written as:
```suggestion
    auto* py_enum_class = PyObject_CallMethod(py_enum_module, "IntEnum", "sO", name, py_constants_dict);
```

Reply:
I'll put the type here as I don't think that it is obvious what type it will be.

Reply:
It's prepended in the python call `PyObject_`, but I'm ok if you want to have it anyway :)